src/components/LeftChat.tsx

```diff
@@ -0,0 +1,98 @@
+import { getDateString } from "../utils/getDateString";
+import styled from "styled-components";
+
+interface LeftChatProps {
+  imgSrc: string;
+  name: string;
+  message: string;
+  date: string;
+}
+
+const LeftChat = ({ imgSrc, name, message, date }: LeftChatProps) => {
+  return (
+    <Wrapper>
+      <ImgContainer>
+        <img src={imgSrc} alt="profile"></img>
```

Comment:
self closing tag로 쓸 수 있습니다!
```suggestion
        <img src={imgSrc} alt="profile" />
```

Reply:
prettier에 기댄 흔적이 남은 코드네요.. 😅
다음 과제에서는` empty tag`로 사용해보겠습니다!